packages/devtools/src/composite.ts

```diff
@@ -134,9 +143,57 @@ export type FromModelsParams = CompositeOptions & {
 export class Composite {
   static VERSION = '1.0'
 
-  static create(_params: CreateParams): Promise<Composite> {
-    // TODO: convert the schema to models, create the models on the Ceramice node, load their commits
-    throw new Error('Not implemented')
+  static async create(params: CreateParams): Promise<Composite> {
+    const { models } = compositeModelsAndCommonEmbedsFromGraphQLSchema(params.schema)
+
+    const modelsMapping: Record<string, ModelDefinition> = {}
+    let commits: Record<string, any> = {}
+
+    await Promise.all(
+      // For each model definition...
+      models.map((modelDefinition) => {
+        // create the model stream,
+        return ModelStream.create(
+          params.ceramic,
+          {
+            name: modelDefinition.name,
+            description: modelDefinition.description,
+            schema: modelDefinition.schema,
+            accountRelation: modelDefinition.accountRelation,
+          },
+          params.metadata
+        ).then((modelStream) => {
+          // put the model stream in the models mapping,
+          modelsMapping[modelStream.id.toString()] = modelDefinition
+        })
+      })
+    )
+
+    await Promise.all(
```

Comment:
This could be folded into the logic applied above for each model, no? Create the model then retrieve commits based on the resulting ID?